core/src/main/scala/io/finch/internal/FutureToEffect.scala

```diff
@@ -0,0 +1,28 @@
+package io.finch.internal
+
+import cats.effect.Effect
+import com.twitter.util.{Future => TwitterFuture, Return, Throw}
+import io.finch.ToEffect
+import scala.concurrent.{Future => ScalaFuture}
+import scala.util.{Failure, Success}
+
+trait FutureToEffect {
+  implicit def twFutureToEffect[E[_]: Effect]: ToEffect[TwitterFuture, E] = new ToEffect[TwitterFuture, E] {
```

Comment:
I wonder if we can `@deprecate` those parameters to encourage users to upgrade their codebase to lawful effects.

Reply:
Maybe we don't have to? Maybe deprecating these wasn't the good idea. These converters are pretty slim and going from Future to Effect turned out to be pretty straightforward. I'm thinking to keep them as is and revisit it later, after 1.0 or after we rework our mapper entirely.

Reply:
That's what I was thinking